[quant][pt2] Add test for inplace add

andrewor14 opened the pull request:
Summary: This was broken after the recent partitioner refactors.

Test Plan: python test/test_quantization.py TestQuantizePT2E.test_qat_inplace_add_relu

Differential Revision: D46402378

jerryzh168 reviewed (approved):
thanks, I remember added this in one of my diffs as well

andrewor14 committed:
[quant][pt2] Add test for inplace add (#102867)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/102867

This was broken after the recent partitioner refactors.

Test Plan: python test/test_quantization.py TestQuantizePT2E.test_qat_inplace_add_relu

Reviewed By: jerryzh168

Differential Revision: D46402378

fbshipit-source-id: f08094a7ef50492dc55cda82a7106520024ec284
